rafay/resource_eks_cluster.go

```diff
@@ -2209,7 +2209,10 @@ func expandEKSClusterConfig(p []interface{}, rawConfig cty.Value) *EKSClusterCon
 		return obj
 	}
 	in := p[0].(map[string]interface{})
-	rawConfig = rawConfig.AsValueSlice()[0]
+	var nRawConfig cty.Value
```

Comment:
@sharan-rafay Cannot use a global variable here to handle every `rawConfig` attribute read, because it could be possible that first `rawConfig.isNull()` is false and `nRawConfig` is initialised. Now, if second `rawConfig.Null()` is true then the second flatten will get wrong inputs for `nRawConfig`. Added a suggestion for same below.

Reply:
updated